connect/main.py

```diff
@@ -93,7 +96,7 @@ def connect():
 def line(target, devicename):
     """Connect to line LINENUM via serial connection"""
     cmd = "consutil connect {}".format("--devicename " if devicename else "") + str(target)
-    run_command(cmd)
+    sys.exit(run_command(cmd))
```

Comment:
why not change line 109 as well?

Reply:
This is not console command, I think we need change it in another PR if need.
Currently, I think we only leverage console related command's exit code, I think it is OK to keep other command no change.

Reply:
what do you mean 109 is not console command? it is also consutil.

Reply:
ahh... i see, I will update it

Reply:
This is a undocumented command. I fixed it and updated the document